monitoring/local_monitoring/gpu-demo/no-gpu/README.md

```diff
@@ -0,0 +1,126 @@
+# **Kruize GPU Demo (NO GPU Scenario for testing kruize + instaslice integration)**
+
+This script automates the installation and testing of **InstaSlice**, a **Fake GPU Operator**, and **Kruize** for dynamic GPU resource optimization. It runs an experimental workload, applies changes based on recommendations, and compares resource requests/limits before and after optimization.
+
+---
+
+## **Overview of the Script**
```

Comment:
Please add a pre-requisites section to install - kubectl, helm, go, ginko and anything else required.

Reply:
@chandrams The tool installations are arch, OS and package manager specific, so will it be a good idea to not have the details of how to install them. I'm inclined to list out the required tool required to run the script if they are not installed. 

The reason why I have added the installation steps to `ginkgo` is because it's installed via `go` but not the system's package manager. 

Please let me know your thoughts on this